LayoutTests/platform/wk2/TestExpectations

```diff
@@ -742,6 +742,11 @@ http/tests/resourceLoadStatistics/grandfathering.html [ Pass ]
 http/tests/resourceLoadStatistics/prune-statistics.html [ Pass ]
 http/tests/resourceLoadStatistics [ Pass ]
 
+# Tests rely on internal behavior
```

Comment:
Do these rely on internal behavior or do they rely on behavior introduced in Ventura and iOS 16?

Reply:
Ventura and iOS 16, will fix this comment.